eng/Versions.props

```diff
@@ -3,6 +3,179 @@
   This file defines the versions of external dependencies used by ASP.NET Core.
   This file might be updated by automation.
 
-  TODO: move versions from build/dependencies.props into this location. Depends on https://github.com/aspnet/AspNetCore/issues/4246.
 -->
-<Project />
+<Project>
+  <!-- These versions should be updated by automation. -->
+  <PropertyGroup Label="Automated">
+    <!-- Packages from dotnet/core-setup -->
+    <MicrosoftExtensionsDependencyModelPackageVersion>3.0.0-preview-27219-3</MicrosoftExtensionsDependencyModelPackageVersion>
+    <MicrosoftNETCoreAppPackageVersion>3.0.0-preview-27219-3</MicrosoftNETCoreAppPackageVersion>
+    <MicrosoftDotNetPlatformAbstractionsPackageVersion>3.0.0-preview-27219-3</MicrosoftDotNetPlatformAbstractionsPackageVersion>
+
```

Comment:
I don't think white space is preserved.

Reply:
Sigh.

Reply:
Probably a one-line fix. ...and a day to find the line, three days to discuss the change, and another day to get it approved, merge it, and get a new build. (lol, sorry--feeling pessimistic today)